stats.c

```diff
@@ -222,6 +227,10 @@ static void error(const char *format, ...);
 int is_in_regions(bam1_t *bam_line, stats_t *stats);
 void realloc_buffers(stats_t *stats, int seq_len);
 
+static int regions_lt(const void *r1, const void *r2) {
+    int from_diff = ((pos_t *)r1)->from - ((pos_t *)r2)->from;
```

Comment:
pos_t->from is uint32_t.  Therefore subtracting any two values would be potentially 33bits long.  Maybe this should be `int64_t diff` to accommodate that, and any realistic future change where we make chromosomes more than 32-bit long.